src/index.html

```diff
@@ -0,0 +1,10 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+	<meta charset="UTF-8">
+	<title>Document</title>
+</head>
+<body>
+	Hello world
+</body>
+</html>
```

Comment:
the document must end with one empty line
```suggestion
</html>

```